common/live-streaming.md

```diff
@@ -84,3 +85,25 @@ For custom apps, you can enable the live streaming destination manually:
 1. Ensure that you have roles for the broadcaster (who can publish their audio/video) and the viewer (who cannot publish audio/video)
 
 ### SDK integration
+
+Use the 100ms client-side SDKs to integrate streaming in your application. See code snippets for the web SDK [here](./../features/hls).
+
+### HLS playback for the audience
```

Comment:
it was a conscious decision to not use the term "HLS" as the primary term in the page - instead, it's used as a supporting term for "live stream" in the page. this is because HLS is esoteric: developers who have built with it know what it means. i know our current API uses the term, but that's going to change

let's be empathetic to a broader user base and call this "Live stream playback" (also drop the "audience" so that the heading is more succinct for easier scanning)

Reply:
fixed it.